packages/syft/src/syft/service/blob_storage/util.py

```diff
@@ -0,0 +1,22 @@
+# stdlib
+from typing import Any
+
+# relative
+from ...util.util import get_mb_serialized_size
+from ..metadata.node_metadata import NodeMetadata
+from ..metadata.node_metadata import NodeMetadataJSON
+
+
+def min_size_for_blob_storage_upload(metadata: NodeMetadata | NodeMetadataJSON) -> int:
+    if not isinstance(metadata, (NodeMetadata | NodeMetadataJSON)):
```

Comment:
Do we need this check ?

Reply:
yeah actually we don't need this. Thanks Shubham